tests/Type/Doctrine/Query/QueryResultTypeWalkerTest.php

```diff
@@ -1325,26 +1246,6 @@ public function getTestData(): iterable
 			',
 		];
 
-		/*yield 'sqrt function' => [
```

Comment:
I don't see this commented out test added again. It's commented out because it fails on some versions, but it's still valuable to re-add it at some point in the future.